src/Microsoft.ML.SearchSpace/Microsoft.ML.SearchSpace.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
+    <LangVersion>9.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.ML.ModelBuilder.SerializablePipeline.Tests" />
+    <InternalsVisibleTo Include="Microsoft.ML.ModelBuilder.SearchSpace.Tests" />
+    <InternalsVisibleTo Include="Microsoft.ML.ModelBuilder.SerializablePipeline" />
```

Comment:
Do you still need `ModelBuilder` in these? And for now can you remove the pipeline stuff? I don't see those projects in this pr. Unless I am missing them somewhere.

Reply:
### Resolved